api/opds.py

```diff
@@ -721,6 +721,10 @@ def language_and_audience_key_from_work(self, work):
             audiences = Classifier.AUDIENCES_JUVENILE
         elif work.audience in Classifier.AUDIENCES_ADULT:
             audiences = list(Classifier.AUDIENCES)
+        elif work.audience == Classifier.AUDIENCE_ALL_AGES:
+            audiences = [Classifier.AUDIENCE_ALL_AGES]
+        elif work.audience == Classifier.AUDIENCE_RESEARCH:
+            audiences = [Classifier.AUDIENCE_RESEARCH]
```

Comment:
And this should probably be AUDIENCES, since if you're doing research you probably want to see everything this person wrote.